src/selectors/ariaAttributes.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ariaAttributes
+ */
+import shallowEqualSelector from 'shallowEqualSelector';
+
+/**
+ * Calculate the aria attributes for the rows and the grid.
+ *
+ * @param {number} rowsCount
+ * @param {boolean} useGroupHeader
+ * @param {boolean} useFooter
+ * @return {{
+ *   ariaGroupHeaderIndex: number,
+ *   ariaHeaderIndex: number,
+ *   ariaFooterIndex: number,
+ *   ariaRowCount: number,
+ *   ariaRowIndexOffset: number
+ * }}
+ */
+function calculateAriaAttributes(rowsCount, useGroupHeader, useFooter) {
```

Comment:
@Tanner-MS , can you review this function? I just copied your PR for master, and renamed/rearranged a bit.

Reply:
Looks good